instrumentation/resque/lib/opentelemetry/instrumentation/resque/patches/resque_job.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Resque
+      module Patches
+        # Module to prepend to Resque::Job for instrumentation
+        module ResqueJob
+          def perform # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/CyclomaticComplexity
+            job_class = payload_class_name
+
+            attributes = {
+              'messaging.system' => 'resque',
+              'messaging.destination' => queue.to_s,
+              'messaging.destination_kind' => 'queue',
+              'messaging.resque.job_class' => job_class
+            }
+
+            span_name = case config[:span_naming]
+                        when :queue then "#{queue} process"
+                        when :job_class then "#{job_class} process"
+                        else "#{queue} process"
+                        end
```

Comment:
Just curious: why do you prefer duplicating the `:queue` and default cases rather than:
```suggestion
            span_name = config[:span_naming] == :job_class ? "#{job_class} process" : "#{queue} process"
```
or
```suggestion
            span_name = case config[:span_naming]
                        when :job_class then "#{job_class} process"
                        else "#{queue} process"
                        end
```

Reply:
>Just curious: why do you prefer duplicating the :queue and default cases rather than

Less about duplication and more about the explicitness of it.  The alternative is cleaner though, I'll commit that 👍